updateQDKVersion.ps1

```diff
@@ -0,0 +1,61 @@
+$ver = $args[0]
+Write-Host $ver
+
+$csString = 'PackageReference Include=\"Microsoft\.Quantum\.[a-zA-Z\.]+\" Version=\"(?<oldVersion>[0-9|\.]+)\"'
+$csFiles = (Select-String -Path "**\*.csproj" -pattern "Microsoft.Quantum" | Select-Object -Unique Path)
+$csFiles | ForEach-Object {
+    (Get-Content -Encoding UTF8 $_.Path) | ForEach-Object {
+         $isQuantumPackage = $_ -match $csString
+         if ($isQuantumPackage) {
+             $_ -replace $Matches.oldVersion, $ver
+         } else {
+             $_
+         }
+    } | Set-Content -Encoding UTF8 $_.Path
+}
+
+$ipynb_string = '%package Microsoft.Quantum.Katas::(?<oldVersion>[0-9|\.]+)'
+$ipynbFiles =  (Select-String -Path "**\*.ipynb" -pattern "Microsoft.Quantum" | Select-Object -Unique Path)
+
+$ipynbFiles | ForEach-Object {
+    if ($_)
```

Comment:
Why do we need this check? I thought none of the results we loop over should be empty?..

Reply:
So if we do go the recurse way, there might be ipynb that do not have a match. I ran into that problem before, so this is just a catch-all.